vendor/licenses/git_submodule/wa-tmLanguage.dep.yml

```diff
@@ -0,0 +1,671 @@
+---
+name: wa-tmLanguage
+version: bf5c4f06c9e2abef53c1f6bbe4722dfd10fee39d
+type: git_submodule
+homepage: https://github.com/wa-lang/wa-tmLanguage.git
+license: agpl-3.0
```

Comment:
Please run this to update the cached license:

https://github.com/github-linguist/linguist/blob/6fb4fb19f8931bfacabfb78b3acb203811c3cd98/script/add-grammar#L135